cosmosdb/client_endpoint.bal

```diff
@@ -267,12 +276,13 @@ public client class DataPlaneClient {
         return mapJsonToStoredProcedure(jsonResponse);
     }
 
-    # List information of all stored procedures.
+    # Lists information of all stored procedures.
     #
     # + databaseId - ID of the database to which the container belongs to
     # + containerId - ID of the container which contains the stored procedures
-    # + resourceReadOptions - The `ResourceReadOptions` which can be used to add additional capabilities to the request
-    # + return - If successful, returns a `stream<StoredProcedure, error>`. Else returns `Error`.
+    # + resourceReadOptions - The `cosmos_db:ResourceReadOptions` which can be used to add additional capabilities to 
+    #                         the request
+    # + return - If successful, returns a `stream<cosmos_db:StoredProcedure, error>`. Else returns `cosmos_db:Error`.
```

Comment:
Suggestion: "`stream<cosmos_db:StoredProcedure, error>`, or else the `cosmos_db:Error` error"